tests/connection/broker.cpp

```diff
@@ -96,7 +96,8 @@ class NamedSubscriber : public Subscriber
                 return BrokerPtr(
                     new zeq::connection::Broker(
                       "zeq::connection::test_named_broker", subscriber, mode ));
-                lunchbox::sleep( 100 );
+                // cppcheck-suppress unreachableCode
```

Comment:
cppcheck is right here, no? there is a return statement before, so why would that 'fail'?